binary_search_tree/binary_search_tree.py

```diff
@@ -30,7 +30,31 @@ def insert(self, value):
           complete = True
 
   def contains(self, target):
-    pass
+    current = self
+    complete = False
+
+    while not complete:
+      #if there is nothing left to check, then return false
+      if not current:
+        return False
+      #if the value of the current node is equal to the target
+      if current.value == target:
+        return True
+      #if the current value is greater than the target
+      elif current.value > target:
+        current = current.left
+      #if current value is less than the target 
+      else:
+        current = current.right
 
   def get_max(self):
-    pass
+    current = self
+    max = 0
```

Comment:
you can also just check for the right-most node. since you know that any item to the right of a node must be larger than it, the fact that it exists means it's larger than it's parent - no need to keep a curr_max.